src/extension/notifier/src/main/resources/applicationContext.xml

```diff
@@ -4,6 +4,8 @@
     <bean id="notifierProcess" class="au.org.emii.wps.NotifierProcess">
         <constructor-arg index="0" ref="wpsResourceManager"/>
         <constructor-arg index="1" ref="httpNotifier"/>
+        <constructor-arg index="2" value="3" />
+        <constructor-arg index="3" value="300000" />
```

Comment:
You'll just need to have a setter and getter for each. There also must be an annotation for that. If there *is* an annotation for that, I'll change GGD and ncwms accordingly.